.github/workflows/python_sdk_publish.yaml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
```

Comment:
I don't think our current Python SDK testing process is strict enough for this to run on every push right now. Could we make this work on workflow_dispatch instead for now?